eotsmanager/cmd/eotsd/daemon/sign.go

```diff
@@ -59,6 +59,67 @@ var SignSchnorrSig = cli.Command{
 	Action: SignSchnorr,
 }
 
+var VerifySchnorrSig = cli.Command{
+	Name:      "verify-schnorr-sig",
+	Usage:     "Verify a schnorr signature over arbitrary data.",
+	UsageText: "verify-schnorr-sig [file-path]",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:     fpPkFlag,
+			Usage:    "The public key pair of the private key that signed the file",
```

Comment:
```suggestion
			Usage:    "The EOTS public key that will be used to verify the signature",
```
This command is agnostic to whether it is the same key pair signed the file

Reply:
https://github.com/babylonchain/finality-provider/pull/312/commits/ec1c7dba008481cb834c6e0b85d3a766551a6e69